scout/models/phenotype_term.py

```diff
@@ -27,49 +27,3 @@ class HpoTerm(BaseModel):
     def set_hpo_number(self) -> "HpoTerm":
         self.hpo_number = int(self.hpo_id.split(":")[1])
         return self
-
-
-# Disease terms represent diseases collected from omim, orphanet and decipher.
-# Collected from OMIM
-class DiseaseTerm(dict):
-    """Represents a disease term
-
-    _id = str, # Same as disease_id
-    disease_id = str, # required, like OMIM:600233
-    disease_nr = int, # The disease nr
-    description = str, # required
-    source = str, # required
-    genes = list, # List with integers that are hgnc_ids
-    inheritance = list, # List of inheritance models connected to the disease
-    hpo_terms = list, # List of hpo terms associated with the disease
-
-    """
-
-    def __init__(
-        self,
-        disease_id,
-        disease_nr,
-        description,
-        source,
-        genes=None,
-        inheritance=None,
-        hpo_terms=None,
-    ):
-        super(DiseaseTerm, self).__init__()
-        self["disease_id"] = disease_id
-        self["_id"] = disease_id
-        self["disease_nr"] = int(disease_nr)
-        self["description"] = description
-        self["source"] = source
-        self["genes"] = genes or []
-        self["inheritance"] = inheritance or []
-        self["hpo_terms"] = hpo_terms or []
-
-
-# phenotype_term is a special object to hold information on case level
-# This one might be deprecated when we skip mongoengine
-phenotype_term = dict(
```

Comment:
You could drop it in `models/case/case.py` for now. 🤷‍♂️ Its not very informative, but there would be no descriptions of the db dict anywhere if it goes.

Reply:
But it's not a case model, we wouldn't even been looking for it in there. I'll add some description to the pydantic one

Reply:
Just trying to be helpful here: we would want db compatible obj model descriptions for
- [ ] `phenotype_term` 
- [ ] `DiseaseTerm`